devops/ansible/playbook.yml

```diff
@@ -233,6 +233,9 @@
       DISPLAY: :7
     when: arbor
 
+  - name: arbor | install logo
+    get_url: url=https://arbor.kitware.com/arbor.jpg dest=/vagrant/app/arbor.jpg
```

Comment:
Makes sense to me. Not sure how to do the condition in ansible but @zackgalbreath could you give it a try? Perhaps an earlier rule with no `when: ` condition would be adequate - and the arbor one would overwrite it later if `arbor` is `true`. You can use http://www.tangelohub.org/wp-content/uploads/2014/05/Tangelo_Mark_256.png for the Tangelo logo.